src/widget/OktaSignIn.js

```diff
@@ -157,7 +187,7 @@ var OktaSignIn = (function() {
         Router.prototype.Events.on.apply(this, onArgs);
       }
     });
-    _.extend(this, getProperties(authClient, Router, options));
+    _.extend(this, getProperties(authClient, hooks, Router, options));
```

Comment:
can be passed a part of `options` ? `options.hooks = hooks`

Reply:
options.hooks (set in widget config) is used to create the Hooks model which is available before or after render.